modules/utils/version.variant

```diff
@@ -0,0 +1,13 @@
+job "version" {
+  description = "Shows CLI version"
+
+  variable "atmos_version" {
+    type  = string
+    value = "0.16.0"
```

Comment:
```suggestion
    value = "Not implemented yet."
```

Reply:
Why "Not implemented yet"? 
We don't update `atmos` too often and can update the version when we cut a release until we do it in CI.